### ISSUE TITLE
Mismatch in ratios calculation

### ISSUE THREAD

**Issue by itsMyrto** [itsMyrto/CarRacing-v2-gymnasium#1]
In PPO when the ratios are computed it hass: epx(logits - log probs). Although it does not prevent the agent from learning, it should have the same type, either keep logits, or add a softmax layer in the actor network.
